src/components/Avatar/index.tsx

```diff
@@ -41,8 +45,18 @@ export const Avatar = ({
 					{initials}
 				</GrommetAvatar>
 			)}
-			{!initials && !src && (
-				<GrommetAvatar size={emphasized ? 'large' : 'medium'} {...otherProps} />
+			{!initials && !src && children && (
+				<GrommetAvatar size={emphasized ? 'large' : 'medium'} {...otherProps}>
+					{children}
+				</GrommetAvatar>
+			)}
+			{!initials && !src && !children && (
+				<GrommetAvatar size={emphasized ? 'large' : 'medium'} {...otherProps}>
+					<FontAwesomeIcon
```

Comment:
No need to accept children, just default to this if there is no src and initials.

Reply:
it seems limiting IMHO not being able to have a custom element inside the avatar. Why should we avoid a custom element ?

Reply:
@JSReds why would you want a custom element? What else would you put if you neither have initials nor image? It seems completely unnecessary, as anyone would just want an avatar icon in any case

Reply:
A case that comes to mind is when you want to show another icon that is not the default one for example. My only doubt is why we want to limit grommet features (size, Avatar child ...) when we already have them. However I can remove that section without problems, it was just to understand :)

Reply:
@JSReds the reason is it is easier to add props, but it is more difficult to remove them (breaking change, meaning clients will have to make changes as well). We always try to be as constrained and opinionated as possible, and only expand the API surface if truly necessary (and in this case it isn't since we don't have a use-case that requires custom avatar children). This also ensures that we have consistent behavior over all of our projects (all avatar look the same, and are familiar to users).